src/services/traces/traces.service.test.js

```diff
@@ -102,6 +107,14 @@ function postMilestoneTestCases() {
     const trace = await createTrace(createMileStoneData);
     assert.isFalse(trace.verified);
   });
+
+  it('should set userAddress as ownerAddress of trace, doesnt matter what you send', async () => {
```

Comment:
This test case is for making sure that user can not create a trace with another ownerAddress

Reply:
Thanks